README.md

```diff
@@ -4,13 +4,29 @@ This is a command-line tool to take the [FedRAMP](http://www.fedramp.gov/) Syste
 
 ## Usage
 
-Requires [Go](https://golang.org/) 1.6+.
+Requires [Go](https://golang.org/) 1.6+. 
+
 
 1. Install the tool:
 
+    For Ubuntu/Debian:
     ```bash
+    sudo apt-get install libxml2-dev 
     go get github.com/opencontrol/fedramp-templater
     ```
+    
+    For OsX:
+    ```
+    brew install homebrew/dupes/apple-gcc42
+    export CC=/usr/local/Cellar/apple-gcc42/4.2.1-5666.3/bin/gcc-4.2
+    brew install libxml2
+    ln -s /usr/local/Cellar/libxml2/2.9.4/lib/pkgconfig/libxml-2.0.pc /usr/local/lib/pkgconfig/libxml-2.0.pc
+    export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
+    go get github.com/moovweb/gokogiri
+    go get github.com/opencontrol/fedramp-templater
+    ```
```

Comment:
Minor: Mind splitting "install the dependencies" from "install the tool"? Also, mind adding `bash` to the syntax blocks for highlighting?